Infinite reconnect on client side

DBLaci opened the pull request:
First I tought this should be a setting, but I think it is generally a good idea to reconnect always. Without this the client didn't reconnect after laptop sleep mode or internet disconnects - only on openvpn service restart.

What do you think? If you want I make a configuration value for this.

kyl191 commented:
Hi,
Yes, please make it a config option, and leave the default at 5.
Thanks! :)

DBLaci commented:
I created the new variable, and added an other one (client-to-client), I hope you don't mind, I think it is useful. I updated the readme too. I hope you will like it.
